src/app/news/news.component.html

```diff
@@ -1 +1,16 @@
-<p>news works!</p>
+<div class="container">
+  <div class="card" *ngFor="let card of news">
+    <div class="thumbnailImage">
+      <img [src]="imagesCover" alt="a dummy image" >
+    </div>
+    <div class="thumbnailContent">
+      <div>
+        <h2>{{card.name}}</h2>
+        <p>{{card.description}}</p>
+      </div>
+      <div>
+        <p id="category"><i class='fas fa-tag'></i>  {{card.category}}</p>
```

Comment:
this id exists within an ngFor, which means that we will have the same id for every single card. That is a mistake as each id should be unique on a page. So if you plan to use it you need to ensure that each card's id is unique!